persistence.js

```diff
@@ -525,9 +525,13 @@ function returnSubsForClient (subs) {
   }
 
   for (const subKey of subKeys) {
+    const sub = JSON.parse(subs[subKey])
```

Comment:
Why json parse is needed?

Reply:
Because I also used JSON.stringify and that was because I needed a string instead of a buffer as the original code used the string methods of Redis. Just found out that adding Buffer to the Redis operation names makes them accept/return buffers ;-)